### DISCUSSION TITLE
Broken Arch builds (and others)

### DISCUSSION THREAD

**Discussion by hockeymikey** [hawkeye116477/waterfox-deb-rpm-arch-AppImage#55] [General]
https://build.opensuse.org/package/live_build_log/home:hawkeye116477:waterfox/waterfox-classic-kpe/Arch/x86_64
The Arch builds have been broken for awhile (no idea why, logs don't tell me much as a person who isn't very familiar with building it)

Others seem broken too like CentOS 8 and RockyLinux (but for other reasons).

**Comment by hawkeye116477** [comment]
I know about that. In case of Arch I have no idea what could be wrong. Seems that errors starts after building versions higher than 2022.04. However it works fine when building locally on Manjaro and one person on aur also says that works good on Arch, so it's weird.

In case of Rocky Linux, that's rather something to be fixed by OBS team, cuz some repos aren't working as you see on https://build.opensuse.org/project/show/RockyLinux:8 (red exclamation marks).

In case of CentOS 8, looks like that's some problem with OBS tools, but it's already EOL.

**Reply by hockeymikey** [reply]
No way, you fixed Arch! Thank you, what ended up being the solution?

Makes sense, just thought I'd mention while I was here too.

**Reply by hawkeye116477** [reply]
Not really fixed. I'm testing previous commits to find first bad.

**Reply by hawkeye116477** [reply]
Looks like I found culprit. It's caused by https://github.com/WaterfoxCo/Waterfox-Classic/pull/104 and https://bugzilla.mozilla.org/show_bug.cgi?id=1473291 is solution.
